modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_builder_field_types_modal.js

```diff
@@ -74,6 +74,10 @@ AUI.add(
 
 								var columnNode = instance._createColumn(size);
 
+								var fieldLabel = 'field-type-' + fieldType.get('label');
+
+								fieldType.set('label', Liferay.Language.get(fieldLabel));
```

Comment:
@brunobasto, please, check if these labels are been rendered correctly. For me, the label is showing as: field-type-text, for example.